samples/asana/program.py

```diff
@@ -0,0 +1,53 @@
+"""Demonstration of AutoKitteh's Asana integration.
+
+This script interacts with the Asana API to manage tasks, including creating new ones
+and updating existing tasks.
+"""
+
+import os
+
+from autokitteh.asana import asana_client
+
+import asana
```

Comment:
This is an incorrect import block. We have an automatic check to block this, which is good, but you should have seen this in your IDE, and fixed it automatically by saving the file.

Make sure that the Ruff extension is installed, and configured to consider the repo's Ruff config, not just the IDE's, and to auto-format with Ruff whenever you save the file.